app/navigation/Navigator.scala

```diff
@@ -45,6 +45,7 @@ class Navigator @Inject()() {
     case ReEnterAmountsPage => ua => reEnterAmounts(ua)
     case EmployerContributionPage(year, index) => ua => employerContribution(ua, year, index)
     case ExpensesEmployerPaidPage(year, index) => ua => expensesEmployerPaid(ua, year, index)
+    case Submission => ua => submission(ua)
```

Comment:
Can write this as just
```suggestion
    case Submission => submission
```